core/templates/.spacemacs.template

```diff
@@ -204,8 +204,8 @@ It should only modify the values of Spacemacs settings."
    ;; source settings. Else, disable fuzzy matching in all sources.
    ;; (default 'always)
    dotspacemacs-helm-use-fuzzy 'always
-   ;; If non-nil the paste micro-state is enabled. When enabled pressing `p'
-   ;; several times cycle between the kill ring content. (default nil)
+   ;; If non-nil, the paste transient-state is enabled. And pressing `p' several
+   ;; times, cycles through the elements in the `kill-ring'. (default nil)
```

Comment:
These comments should be a copy of the variable's documentation. See https://github.com/duianto/spacemacs/blob/75026fe22b14869ee6732b01c4785f8d6e3dd06e/core/core-dotspacemacs.el#L252